view-generator-framework/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ dependencies {
   testImplementation("org.junit-pioneer:junit-pioneer:2.0.0")
   testImplementation("org.mockito:mockito-core:5.2.0")
   testImplementation("org.apache.kafka:kafka-streams-test-utils:7.2.1-ccs")
-  testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.20.0")
+  testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j2-impl:2.20.0")
```

Comment:
the api on line 22 is slf4j1, why are we changing to bind the slf4j2 impl?

Reply:
Getting this error with slf4j and moving to slf4j2 resolved this.

```SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:/Users/laxman/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-slf4j-impl/2.20.0/7ab4f082fd162f60afcaf2b8744a3d959feab3e8/log4j-slf4j-impl-2.20.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.
```